src/libs/brands.ts

```diff
@@ -24,14 +24,52 @@ import terraLogoShadow from 'src/images/brands/terra/logo-wShadow.svg'
 const nonBreakingHyphen = '\u2011'
 
 
+export interface BrandConfiguration {
+  /** Brand name */
+  name: string
+
+  /** Used to construct return URLs for FireCloud */
+  queryName: string
+
+  /** Landing page header text */
+  welcomeHeader: string
+
+  /** Landing page text */
+  description: string
+
+  /** Host name for branded site */
+  hostName: string
+
+  /** Links shown on landing page */
+  docLinks: {
+    /** Link URL */
+    link: string
+
+    /** Link text */
+    text: string
+  }[]
+
+  /** URLs for logo images */
+  logos: {
+    /** Brand logo */
+    color: string
+
+    /** Light version of brand logo used against dark backgrounds */
+    white: string
+    [otherLogoType: string]: string
+  }
+
+  /** Optionally filter which datasets show up in the Data Catalog */
+  catalogDataCollectionsToInclude?: string[]
+}
+
 /**
  * Configuration for Terra co-brands (a.k.a. white label sites)
  * https://broadworkbench.atlassian.net/wiki/spaces/WOR/pages/2369388553/Cobranding+and+White+Label+Sites
  */
-export const brands = {
+export const brands: { [brandName: string]: BrandConfiguration } = {
```

Comment:
Record<string, BrandConfiguration> is a nicer way to specify the type as "dictionary object of BrandConfiguration values"

Reply:
I waffled a bit between this and `Record`, but I also like that this notation lets you provide some information about what the keys represent (brand name in this case).

Reply:
🤔 Though I suppose the ideal might look something more like:

```ts
type BrandName = 'brand1' | 'brand2' | ...
const brands: Record<BrandName, BrandConfiguration> = { ... }
```

Reply:
Yeah, I also lean towards the `Record` here, personally.